dockerHashing/src/main/scala/cromwell/docker/registryv2/DockerRegistryV2Abstract.scala

```diff
@@ -27,19 +27,25 @@ object DockerRegistryV2Abstract {
   }
 
   val DigestHeaderName = CaseInsensitiveString("Docker-Content-Digest")
-  val ManifestV2MediaType = "application/vnd.docker.distribution.manifest.v2+json"
-  val ManifestListV2MediaType = "application/vnd.docker.distribution.manifest.list.v2+json"
+  val DockerManifestV2MediaType = "application/vnd.docker.distribution.manifest.v2+json"
+  val DockerManifestListV2MediaType = "application/vnd.docker.distribution.manifest.list.v2+json"
+  val OCIIndexV1MediaType = "application/vnd.oci.image.index.v1+json"
```

Comment:
Nice to have - link to the spec we got these from. Somewhat mysterious without context.